index.html

```diff
@@ -0,0 +1,522 @@
+            <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+            <a href="#" class="year"> 2015 </a>
+          </div>
+        </div>
+        <div class="work-detail">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="tags">
+          <li class="tag-container">
+           <a href="#" class="tag">html </a>
+          </li>
+          <li class="tag-container">
+            <a href="#" class="tag">css</a>
+          </li>
+          <li class="tag-container">
+           <a href="#" class="tag tag-javascript">javascript</a>
+          </li>
+        </ul>
+        <div class="action">
+          <button type="button" class="btn"><span>See Project</span> </button>
+        </div>
+      </div>
+    </div>
+    <!---------Ecnd of ard-works for Tonic-2-->
+    <!---------Card-works for multi-post stories-2-------->
+    <div class="card-works multipost-stories-2-mobile multipost-card">
+      <img src="./images/Snapshoot-Portfolio-4.png" alt="Portfolio Snapshot-4" class="card-work">
+      <div class="left-block">
+      <div class="project">
+        <div class="project-title">                         
+          <a href="#" class="title">Multi-purpose stories</a>
+        </div>
+        <div class="frame-2">
+          <a href="#" class="canopy btn-link"> Canopy</a>
+          <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+          <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+          <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+          <a href="#" class="year btn-link"> 2015 </a>
+        </div>
+      </div>
+      <div class="work-detail">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <ul class="tags">
+        <li class="tag-container">
+         <a href="#" class="tag">html </a>
+        </li>
+        <li class="tag-container">
+          <a href="#" class="tag">css</a>
+        </li>
+        <li class="tag-container">
+         <a href="#" class="tag tag-javascript">javascript</a>
+        </li>
+      </ul>
+      <div class="action">
+        <button type="button" class="btn"><span>See Project</span> </button>
+      </div>
+      </div>
+    </div>
+        <!---------End of caard-works for multi-post stories-2------------->
+        <!----end of container div------------->
+  </section>
+    <!-----section-work ends here------->
+
+    <!---------About myself section------->
+    <section id="about" class="section-about-myself background-color">
+      <div class="about-myself">
+        <div class="top">
+          <div class="welcome">
+            <h2 class="name-welcome">
+               About <br> Myself
```

Comment:
- [ ] please fix the about me section so i looks like the figma design.

your design
![Screenshot (392)_LI](https://user-images.githubusercontent.com/65298048/187800289-ab454882-7184-4d8c-9513-671be83ce2f3.jpg)
figma design

![Screenshot (393)](https://user-images.githubusercontent.com/65298048/187800371-36ea9f2a-5441-4e0b-bdfa-311234b3109e.png)
![Screenshot (393)](https://user-images.githubusercontent.com/65298048/187800660-223c69e6-c25b-40a2-a3f5-1db05ba12bf1.png)